diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -62,6 +62,8 @@ fun ASTNode.hasChildOfType(elementType: IElementType): Boolean =
 fun ASTNode.hasAnyChildOfTypes(vararg elementType: IElementType): Boolean =
         elementType.any { this.hasChildOfType(it) }
 
+fun ASTNode.isBlockEmpty(node: ASTNode?) = node?.let { listOf(ElementType.LBRACE, ElementType.RBRACE, WHITE_SPACE)
```

Comment:
Why do you pass a node as a parameter, when in extension method you should use `this`?